### DISCUSSION TITLE
Roms not showing?

### DISCUSSION THREAD

**Discussion by Dvdmike** [OnionUI/Onion#1411] [Q&A] [answered]
Hi all, new to the MMP and installed Onion using the step by step guide and copied over the bios and ROM files and added my own roms also.
I tried refreshing and no luck.
I looked for the .XML lists on the original card that came with it, no luck.
I then used the beta build last night and no luck again, still cannot get a roms list at all, but going into retroid I can see and play the games.
I am getting very frustrated can anyone help?

**Comment by Aemiii91** [comment]
Can you please provide some screenshots of your rom folders?

**Reply by Dvdmike** [reply]
<img width="479" alt="Screenshot 2024-02-07 082129" src="https://github.com/OnionUI/Onion/assets/159237660/553bd898-7d43-41ec-b2d4-f8a107a0e228">
![Uploading Screenshot 2024-02-07 082153.png…]()

**Reply by Dvdmike** [reply]
<img width="446" alt="Screenshot 2024-02-07 082153" src="https://github.com/OnionUI/Onion/assets/159237660/4461cea0-078f-49aa-9426-4263698c175f">

**Reply by Aemiii91** [reply]
I need to see the rom file structure

**Reply by Dvdmike** [reply]
<img width="621" alt="Screenshot 2024-02-07 082844" src="https://github.com/OnionUI/Onion/assets/159237660/84314463-f191-4816-809c-d3ae0eadaaf8">

all were copied 1 for 1 from the included SD card

**Reply by Aemiii91** [reply]
And the Games -> GBA section is completely empty?

**Reply by Dvdmike** [reply]
If I turn it on with the onion card in, I click on games and it's blank, select and refresh still leaves it blank.
I feel like I missed a step.

**Reply by Aemiii91** [reply, marked as answer]
Sounds like you might not have enabled the systems in Package Manager

**Reply by Dvdmike** [reply]
That was it! Thanks 😊
Now I have another issue that half my PS1 content isn't there but work in retroarch, and I have the bios for sega CD but they don't run after trying to load and the same with Atari lynx.
